firefox-ios/Client/Frontend/PasswordManagement/AddCredentialViewController.swift

```diff
@@ -154,6 +158,9 @@ extension AddCredentialViewController: UITableViewDataSource {
             loginCell.configure(viewModel: cellModel)
             loginCell.applyTheme(theme: themeManager.currentTheme)
             usernameField = loginCell.descriptionLabel
+            if isRTLLanguage {
```

Comment:
I think we can use [`.natural`](https://developer.apple.com/documentation/uikit/nstextalignment/natural) instead ? What do you think? This would avoid us from having a `UIApplication.shared.userInterfaceLayoutDirection == .rightToLeft` check

Reply:
That's a great idea. I did tried to see in debugging, it was `.natural` for both website and username field by default and also tried to use `.natural`, but it was not working. 

Here is image with `.natural` for website Field
<img width="283" alt="image" src="https://github.com/mozilla-mobile/firefox-ios/assets/59967463/4f0d8f4b-ddaa-4ed3-ade4-ff4217512298">

Reply:
Ok, I would need to check why this isn't working, but first thought for sure is that it feels like the `lazy var descriptionLabel: UITextField = .build` `LoginDetailTableViewCell` should be adjusted so all textfields based on it would benefit from the improvement (rather than making per section i.e. one change for usernameField, one change for websiteField, etc.) Password field should be adjusted for RTL as well no?

Reply:
Well, you are right @lmarceau. I am also not in favour to use above solution if we can find something robust. 

For **password** :
I am not sure how but while debugging for field **website, username** in descriptionLabel the style was `.natural` but for password it was `.right. and if you remove `.isSecureTextEntry`, for password Field Text will show on left side. 

I also think `isSecureTextEntry` has something that make it work for Password but not for other field.

Reply:
I had a quick look, and I think setting `label.textAlignment = .natural` under the `descriptionLabel: UITextField = .build` is the way to go. There's only one issue, which is when we add a password and click on the textfield, the field is moved to be left aligned. 

https://github.com/mozilla-mobile/firefox-ios/assets/11338480/36552a42-e602-4200-b51e-ea4d2d8375e2

It could be caused by constraints issues as I do see warnings on those fields for that. 

![Screenshot 2024-02-23 at 11 45 53 AM](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/1bf763f4-a3b8-4549-8f57-1b87beda9e23)

I didn't see any issue with the password fields 🤔 

![Simulator Screenshot - iPhone 15 - 2024-02-23 at 11 40 06](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/1ce133ad-7180-4224-93fe-922b7acc3ec9)

![Simulator Screenshot - iPhone 15 - 2024-02-23 at 11 51 12](https://github.com/mozilla-mobile/firefox-ios/assets/11338480/0a08d445-9ade-41aa-8e05-0a51958d6816)

Reply:
Ah ok I see! Thanks for the explanation, in that case I think we can merge as is 👍

Reply:
Thanks @lmarceau !